package.json

```diff
@@ -30,8 +30,9 @@
     "postinstall": "npm run build",
     "start": "wds-start",
     "theme": "run-s theme:pot theme:icons",
+    "theme:node": "run-s theme:icons",
```

Comment:
Do we really need `theme:node`? With this change, its only purpose is to run `theme:icons` and if someone wants to run that they can just do `npm run theme:icons`.  cc @gregrickaby @jrfoell

Reply:
Or is `theme:node` specific to our Buddy workflow? Maybe it's to future proof for anything additional that may be added to `theme:node` down the road?

Reply:
It's specifically for our Buddy workflow. We keep the normal build so the general public can build "all the things" with one command